integration/images/private-workdir/Makefile

```diff
@@ -0,0 +1,27 @@
+# Copyright The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+all: build
+
+PROJ=gcr.io/k8s-cri-containerd
+VERSION=1.0
+IMAGE=$(PROJ)/private-workdir:$(VERSION)
+
+build:
+	docker build -t $(IMAGE) .
+
+push:
+	gcloud docker -- push $(IMAGE)
```

Comment:
Suggest adding a todo or open an issue to convert to using test images via local image cache... vs round tripping them through gcr.io. Would also be nice to set the test image name via make so we don't have to hard code it in the source file if possible.